javascript/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "piranhaJS",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
@anag004 Shouldn't this be `1.0.1`?

Reply:
It doesn't really matter since we haven't released PiranhaJS as a package (yet). IMO, the best thing to do would be to keep it at `1.0.0`. Let me know if that is fine.

Reply:
fine with me.